packages/gatsby-theme-newrelic/src/components/Collapser.js

```diff
@@ -1,16 +1,23 @@
-import React, { useState, useMemo } from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import Icon from './Icon';
 import { animated, useSpring } from 'react-spring';
 import { usePrevious, useIsomorphicLayoutEffect } from 'react-use';
 import useKeyPress from '../hooks/useKeyPress';
+import useQueryParams from '../hooks/useQueryParams';
 
 const ResizeObserver = global.ResizeObserver || class ResizeObserver {};
 
 const Collapser = ({ title, id, defaultOpen, children }) => {
+  const { queryParams } = useQueryParams();
   const [element, ref] = useState();
-  const [isOpen, setIsOpen] = useState(defaultOpen);
+  const [isOpen, setIsOpen] = useState(() => {
+    return queryParams.has('collapsers') &&
+      queryParams.get('collapsers') === 'openAll'
+      ? true
+      : defaultOpen;
```

Comment:
Out of curiosity, what would you consider to be the correct behavior in this case?

```js
<Collapser defaultOpen={false} />

// url collapsers=openAll
```

Should the collapser be open or closed in this case? I ask because I'm not sure whether the query param should be a fallback or the override. Thoughts?

Reply:
I was thinking the query param is an override to what is otherwise the default (do not open collapsers). i think the most common case is we'll want to collapse them by default unless we have a special need to open them (like supporting smartling contexts)